apps/supernova/src/components/silences/SilenceScheduled.jsx

```diff
@@ -247,7 +259,13 @@ const SilenceScheduled = (props) => {
             <Form>
               <FormSection>
                 <FormRow>
-                  <TextInput required label="Silenced by" value={formState.createdBy} disabled />
+                  <TextInput
+                    required
+                    label="Silenced by"
+                    value={formState.createdBy}
+                    onChange={(e) => onInputChanged({ key: "createdBy", value: e.target.value })}
```

Comment:
You should validate the input here and show an error state for the `TextInput` if the value is empty. 
Also the "Save" button should be disabled as long as the form has errors.

Reply:
already in work